pkg/rpc/context.go

```diff
@@ -1050,18 +1051,32 @@ type delayingHeader struct {
 	DelayMS  int32
 }
 
+func makeDialCtx(
+	dialCtx context.Context, remoteNodeID roachpb.NodeID, target string, class ConnectionClass,
+) context.Context {
+	if remoteNodeID == 0 {
+		dialCtx = logtags.AddTag(dialCtx, "rnode?", nil)
+	} else {
+		dialCtx = logtags.AddTag(dialCtx, "rnode", remoteNodeID)
```

Comment:
I think we should use the standard node formatting here, i.e. `n7` rather than `7`. We can maybe make this more compact by using e.g. `r=n7`.